### PR TITLE
Mission - Fix Bomber function combat behaviour

### PR DESCRIPTION
**Pull request by Mike-MF**
- Sick of this function.
 - AI stand in place, won't do anything and I think it's combat mode doing it.

### CODE REVIEW

File: addons/mission/functions/fnc_bomber.sqf [hunk 1]
```diff
@@ -28,9 +28,10 @@ if (!is3DENPreview && {!isServer}) exitWith {};
 _unit addVest "Umi_Bomb_Vest_Camo";
 
 [QACEGVAR(common,setSpeaker), [_unit, "ACE_NoVoice"]] call CBA_fnc_globalEvent;
+[QGVAR(setCombatBehaviour), [_unit, "CARELESS"]] call CBA_fnc_globalEvent;
```
Comment (jonpas) [inline addons/mission/functions/fnc_bomber.sqf:31]: Should be `targetEvent` at `_unit`. Local Arguments, Global Effects.

Comment (Mike-MF) [reply addons/mission/functions/fnc_bomber.sqf:31]: I did it globally because this could run before HC switch

Comment (jonpas) [reply addons/mission/functions/fnc_bomber.sqf:31]: Does locality switch change combat behaviour? I couldn't find any reference about that.

Comment (Mike-MF) [reply addons/mission/functions/fnc_bomber.sqf:31]: I'd assume so based on how everything else seems to work. It only happens once and probably best to cover the bases.

**Review by jonpas — APPROVED** [review]

**Pull request merged**